packages/main/src/plugin/container-registry.ts

```diff
@@ -1822,6 +1826,16 @@ export class ContainerProviderRegistry {
     }
   }
 
+  async getImageLayers(engineId: string, id: string): Promise<ImageLayer[]> {
+    const tmpdir = fs.mkdtempSync(path.join(os.tmpdir(), 'podman-desktop'));
+    const tarFile = path.join(tmpdir, id + '.tar');
+    await this.saveImage(engineId, id, tarFile);
+    await nodeTar.extract({ file: tarFile, cwd: tmpdir });
+    const result = await getLayersFromImageArchive(tmpdir);
+    fs.rmSync(tmpdir, { force: true, recursive: true });
+    return result;
```

Comment:
probably need a try/finally for deletion so in case of failure we try at least to delete

Reply:
or if we ask two times in 2mn maybe we don't redo the same steps, so maybe it should be scheduled for deletion after some time and use a cache (but can be follow-up)